pondo-bot/.env.example

```diff
@@ -24,3 +24,5 @@ TEST_USER1_PRIVATE_KEY=test_user1_private_key
 TEST_USER1_ADDRESS=test_user1_address
 TEST_USER2_PRIVATE_KEY=test_user2_private_key
 TEST_USER2_ADDRESS=test_user2_address
+LEDGER_PATH=/Users/you/snarkOS/
```

Comment:
Isn't the ledger path whenever you run ./devnet ? So if you start it from within a script here, wouldn't it just place those files wherever and you wouldn't have to worry about copying them or where they are in general?